website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -176,6 +183,52 @@ namespace "default" {
 }
 ```
 
+### Secure Variables
```

Comment:
I've added this block after realizing I also need to add it for the Learn Guide I'm working on in parallel.